lib/hexpm/accounts/email.ex

```diff
@@ -40,6 +40,22 @@ defmodule Hexpm.Accounts.Email do
     |> put_change(:verification_expiry, DateTime.utc_now())
   end
 
+  def changeset(email, :create_for_org, params, false) do
+    cast(email, params, ~w(email public gravatar)a)
+    |> validate_required(~w(email)a)
+    |> update_change(:email, &String.downcase/1)
+    |> validate_format(:email, @email_regex)
+    # TODO: these comments are for code review, need to be removed before merge
+    # |> validate_confirmation(:email, message: "does not match email")
+    # |> validate_verified_email_exists(:email, message: "already in use")
+    |> unique_constraint(:email, name: "emails_email_key")
+    |> unique_constraint(:email, name: "emails_email_user_key")
+    |> put_change(:verified, false)
+
+    # |> put_change(:verification_key, Auth.gen_key())
+    # |> put_change(:verification_expiry, DateTime.utc_now())
```

Comment:
I copy/pasted the body from `def changeset(email, :create, params, verified?) do`, but commented out four function calls that I think are not needed for organization emails.

Let me know if these are okay to be deleted and I'll make a new commit to do so.

Reply:
They can be deleted.

Reply:
deleted in 07e50311de813b127f1ac966ce03551ae7364eed